src/components/header/Header.css

```diff
@@ -193,23 +198,61 @@
     }
 
     .header__menu {
```

Comment:
Nice overlay effect! The header menu is a bit too narrow on mobile though (see for instance 375px), which also distorts the CTA-button. I think it's better to keep the menu full width.

Reply:
yep! i just noticed that, lol. I kept the menu about 60% width for tablet, but at 480px, the menu goes 100% and i fixed the fonts/margins/menu max-min height, so it doesnt look too tall or too short